src/main/scala/uk/gov/hmrc/play/config/GTMConfig.scala

```diff
@@ -27,9 +27,8 @@ class GTMConfig @Inject() (configuration: Configuration) {
     readConfig(_ => "gtm.data.layer.url")
 
   def readConfig(configKey: String => String): Option[String] =
-    configuration.getString("gtm.container").map { container =>
+    configuration.getOptional[String]("gtm.container").map { container =>
```

Comment:
Was removing the re-throw for simplicity? I'd generally be in favour of this but will the default exception generated by a non-existent key be meaningful?

Reply:
Ah let me check it's not introduced something unintended, I can't remember exactly why it was

Reply:
@matthewmascord sorry I forgot to come back to this, the new exception will have a message

> Configuration key 'abc' is set to null but expected string

so I don't think we're losing any useful context compared to the old exception, what do you think?

Reply:
LGTM